### PR TITLE
Add Weijie Guo to committer

### PR DESCRIPTION
**Pull request by reswqa**

### CODE REVIEW

File: docs/community/news.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ license: |
 
 | Date              | Title                                                   | Brief                                                                                                                           |
 |-------------------|---------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------|
-| 2024 November 29  | Release 0.5.2                                           | Celeborn release 0.5.2.                                                                                                         |
+| 2024 November 28  | New Committer: Weijie Guo                               | We are happy to announce Weijie Guo becomes a new Celeborn committer.                                                           |
+| 2024 November 26  | Release 0.5.2                                           | Celeborn release 0.5.2.                                                                                                         |
```
Comment (reswqa) [inline docs/community/news.md:24]: This change(Date of release 0.5.2 from November 29 -> November 26)  has been confirmed with @SteNicholas.

**Comment by turboFei** [comment]
Congrats ~

**Review by turboFei — APPROVED** [review]

**Review by RexXiong — APPROVED** [review]
Congrats ~

**Comment by codenohup** [comment]
Congrats ❤️ @reswqa

**Comment by SteNicholas** [comment]
@reswqa Congrats ❤️

**Comment by waitinfuture** [comment]
@reswqa Congrats!

**Pull request merged**